models/product-tailoring/README.md

```diff
@@ -0,0 +1,241 @@
+// Basic REST models
+const productTailoringRest = ProductTailoringRest.random().build();
+const productTailoringDataRest = ProductTailoringDataRest.random().build();
+const productVariantTailoringRest =
+  ProductVariantTailoringRest.random().build();
+const productTailoringAttributeRest =
+  ProductTailoringAttributeRest.random().build();
+
+// Basic GraphQL models
+const productTailoringGraphql = ProductTailoringGraphql.random().build();
+const productTailoringDataGraphql =
+  ProductTailoringDataGraphql.random().build();
+const productVariantTailoringGraphql =
+  ProductVariantTailoringGraphql.random().build();
+const productTailoringAttributeGraphql =
+  ProductTailoringAttributeGraphql.random().build();
+
+// Presets
+const basicRestPreset = ProductTailoringRest.presets.basic().build();
+const basicGraphqlPreset = ProductTailoringGraphql.presets.basic().build();
+```
+
+# Models
+
+## ProductTailoring
+
+The main model representing a product tailoring. Available in both REST and GraphQL variants.
+
+### Presets
+
+#### Basic Preset
+
+The basic preset provides a complete product tailoring with all essential fields and three variants. It includes:
+
+- Localized fields in English and German:
+
+  - name: "Tailored Product" / "Angepasstes Produkt"
+  - description: "Product tailored for specific market" / "Produkt für spezifischen Markt angepasst"
+  - metaTitle: "Tailored Product - Meta Title" / "Angepasstes Produkt - Meta Titel"
+  - metaDescription: "Meta description for tailored product" / "Meta-Beschreibung für angepasstes Produkt"
+  - metaKeywords: "tailored, product, market" / "angepasst, produkt, markt"
+  - slug: "tailored-product" / "angepasstes-produkt"
+
+- Product state:
+
+  - key: "tailored-product-key"
+  - published: true
+  - hasStagedChanges: false
+
+- Three variants, each with:
+  - Unique ID
+  - Two product images with dimensions
+  - Color attribute (localized)
+  - Size attribute (localized)
+
+Example:
+
+```ts
+// REST
+const basicRest = ProductTailoringRest.presets.basic().build();
+// Creates a product tailoring with REST structure:
+// {
+//   key: 'tailored-product-key',
+//   published: true,
+//   hasStagedChanges: false,
+//   current: {
+//     name: { en: 'Tailored Product', de: 'Angepasstes Produkt' },
+//     // ... other localized fields
+//     variants: [
+//       {
+//         id: 1,
+//         images: [
+//           {
+//             url: '//product-image-1.jpg',
+//             dimensions: { w: 800, h: 600 },
+//             label: 'Product Image 1'
+//           },
+//           // ... second image
+//         ],
+//         attributes: [
+//           {
+//             name: 'color',
+//             value: { en: 'Color 1', de: 'Farbe 1' }
+//           },
+//           {
+//             name: 'size',
+//             value: { en: 'M', de: 'M' }
+//           }
+//         ]
+//       },
+//       // ... variants 2 and 3
+//     ]
+//   },
+//   staged: { /* same structure as current */ }
+// }
```

Comment:
Shouldn't we remove this commented code and the one below?

Reply:
Ok!